index.js

```diff
@@ -43,7 +45,13 @@ module.exports = {
           passed: passed,
           errorMessage: relativePath + ' should pass ESLint.' + errors
         }]);
-      }
+      },
+
+      // Find broccoli-lint-eslint's babel-eslint parser and use it as our default
+      options: {
+        parser: path.dirname(resolve.sync('broccoli-lint-eslint/node_modules/babel-eslint')),
```

Comment:
See https://github.com/ember-cli/ember-cli-qunit/blob/master/index.js#L19-L21 for an example.

In that example, we resolve `ember-qunit` (which is ember-cli-qunit's dependency) then resolve `ember-test-helpers` relative to the location of `ember-qunit` and then once more for klassy (relative to `ember-test-helpers` location).

This is using the `resolve` NPM package.